src/store/ChartTableStore.js

```diff
@@ -45,9 +45,9 @@ export default class ChartTableStore {
         const year = DT.getUTCFullYear();
         const month = DT.getUTCMonth() + 1;
         const day = DT.getUTCDate();
-        const hours = DT.getUTCHours();
-        const minutes = DT.getUTCMinutes();
-        const seconds = DT.getUTCSeconds();
+        const hours = DT.getHours();
+        const minutes = DT.getMinutes();
+        const seconds = DT.getSeconds();
```

Comment:
...why did you change this?

All of our data should be in UTC time.

Reply:
@brucebinary it is already in UTC time, it was double converting it!

Reply:
@MahboobehMohammadi omg surprised our clients didn't spot that. :O

Thanks for spotting this.

if it is already in UTC time, shouldn't the year month and day also be already in UTC time?

Reply:
@brucebinary yes right, but arent they the same? i mean when does UTC year could be different from local year?! or month.

Reply:
@MahboobehMohammadi At the end of each year. Different parts of the world reach the new year at different times.